src/nlp/metric.py

```diff
@@ -228,18 +231,20 @@ def compute(self, predictions=None, references=None, timeout=120, **metrics_kwar
         predictions = self.data["predictions"]
         references = self.data["references"]
         with temp_seed(self.seed):
-            output = self._compute(predictions=predictions, references=references, **metrics_kwargs)
+            output = self._compute(predictions=predictions, references=references, **kwargs)
```

Comment:
Here we should be careful that `_compute` doesn't allow `**kwargs` otherwise we could have a silent error where for instance `predictions` is written as `prediction` and is not recognized and this goes undetected and the predictions are missed.

This could maybe be done with a test on signature of the metrics `_compute` method in the tests for the metrics scripts (or maybe there is another smart way to do that).

Reply:
I added a test for the signature of `_compute` to make sure it doesn't have `**kwargs`